client/src/redux/sagas/fixtures/notificationsSetupEvent.fixture.js

```diff
@@ -0,0 +1,64 @@
+// a single step is processing
+export const setupEventProcesssingFixture = {
+  event_type: 'setup_event',
+  setup_event: {
+    username: 'maxmustermann',
```

Comment:
The username on the front end probably needs to match an existing user in the python fixtures, for future purposes. (Like, if we ever did some type of integration test between the client sending events to the backend regarding a user)

Reply:
good point!  Auf wiedersehen Max Mustermann.